src/scf-helper-release/jobs/global-properties/spec

```diff
@@ -11,23 +11,23 @@ packages: []
 
 properties:
 
-  hcf.cats-suites:
+  scf.cats-suites:
     description: The list of Cloud Foundry acceptance test suites to run
     example: =override,suites -ignore,suite +additional,suite
 
-  hcf.internal-ca-cert:
+  scf.internal-ca-cert:
     description: "Internal CA certificate for use across the cluster"
 
-  hcf.skip_cert_verify_external:
+  scf.skip_cert_verify_external:
     description: "Skip certificate validation when talking to external endpoints"
 
-  hcf.hcp_service_domain_suffix:
+  scf.hcp_service_domain_suffix:
```

Comment:
I'm not exactly clear on what happened to this config variable, but I think it might have become `kube_service_domain_suffix`?

Reply:
Possibly. Will check.

Reply:
Definitely not used anymore. Removed.
Handled with commit 24d6b772a9682b33b75c02025b54c6a862d51f89